RELEASE-NOTES.md

```diff
@@ -1,7 +1,13 @@
 # About this release
 
-Changes:
-* Add [small localnet](docs/docker.md#localnet) docker-compose configuration for testing and development purposes
+[This release](https://github.com/aeternity/epoch/releases/tag/v0.7.0) is focused on smart contracts: it provides HTTP API for compiling and running smart contract and creating contract transactions.
+It also:
```

Comment:
As I understand, the contracts are only executed _off-chain_, so it makes sense to add that: `Currently only off-chain execution of smart contracts is possible. On-chain execution is planned for the next release.`

Reply:
As discussed, on/off chain already mentioned in intended usage of API linked from release notes.

([Exact page](https://github.com/aeternity/protocol/blob/epoch-v0.7.0/epoch/api/contract_api_usage.md).)